### PR TITLE
feat(unlock-app) - fixing user creation

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
- Fix user creation issue on `POST` endpoint
- Replace Input/Button components and refresh UI with new design

# Issues

Fixes #
Refs #
Screenshot 2022-08-30 at 10 05 48

Screenshot 2022-08-30 at 10 11 26

Screenshot 2022-08-30 at 10 12 00

Screenshot 2022-08-30 at 11 03 03

Screenshot 2022-08-30 at 10 52 21

Screenshot 2022-08-30 at 11 03 36

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
💅

File: unlock-app/src/hooks/useAccount.ts [hunk 1]
```diff
@@ -94,21 +95,31 @@ export const useAccount = (address: string, network: number) => {
     let recoveryKey
 
     try {
-      const response = await storageService.createUser(
+      const errorCodes = [500, 401, 404, 400]
+      const successCodes = [200, 201, 204]
```
[the remark this answers is not in the document]
Comment (julien51) [reply unlock-app/src/hooks/useAccount.ts:99]: Agreed this is very fragile, what happens on 403 or 500?

**Review by julien51 — CHANGES_REQUESTED** [review]
This is looking very good! 
Please fix the HTTP request check and maybe center terms as @searchableguy suggested!

**Comment by kalidiagne** [comment]
Content centred, and added styles also for Login page

Screenshot 2022-08-30 at 17 43 47

Screenshot 2022-08-30 at 17 52 15

Screenshot 2022-08-30 at 17 57 38

**Comment by julien51** [comment]
Also please don't do buttons for "Login" and "connect it" or "cancel"... they look really strange

**Comment by kalidiagne** [comment]
Asked help to @ChialiTsai for new design so i can update it all with new UI

**Comment by kalidiagne** [comment]
- Resized text with better hierarchy
- Remove button for "Login" and "connect it" or "cancel"
- Add more space between form fields and titles 

Screenshot 2022-09-01 at 10 47 00

Screenshot 2022-09-01 at 10 47 08

Screenshot 2022-09-01 at 10 49 02

**Review by julien51 — APPROVED** [review]

**Pull request merged**